Docker/build/WDQS-frontend/Dockerfile

```diff
@@ -13,8 +13,8 @@ COPY --from=fetcher /wikidata-query-gui /tmp/wikidata-query-gui
 
 WORKDIR /tmp/wikidata-query-gui
 
-# Put wdqs gui in the right place
-RUN apk --no-cache add --virtual build-dependencies ca-certificates~=20191127-r2 git~=2.24 nodejs~=12 npm~=12 jq~=1.6 python~=2.7 make~=4.2 g++~=9.3
+# install dependencies
+RUN apk update && apk --no-cache add --virtual build-dependencies ca-certificates git nodejs npm jq~=1.6 make g++
```

Comment:
We should probable be consistent about this and maybe just unpin (or even bump) the ca-certificates

Reply:
ok, dropped the certificate but still had to bump the other stuff

Reply:
gotcha, I'm not really understanding why it seems like bumping he ca-certificates in then breaking the other dependencies. For example, why do we we have to upgrade node versions?

Reply:
Also, I'm not sure I understand the need for `apk update` ? My reading of https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache is that by using `--no-cache` we still get the latest versions.

Reply:
grand job